test/middleware.test.js

```diff
@@ -2663,7 +2864,30 @@ describe('middleware', () => {
         .expect(200, done);
     });
   });
+  describe('headers option with other framework', () => {
+    beforeAll((done) => {
+      const compiler = getCompiler(webpackConfig);
+
+      instance = middleware(compiler, {
+        headers: { 'X-nonsense-1': 'yes', 'X-nonsense-2': 'no' },
+      });
+
+      app = fastify();
+      app.use(instance);
 
+      listen = listenShorthand(done);
+    });
+
+    afterAll(close);
+
+    it('should return the "200" code for the "GET" request to the bundle file and return headers', (done) => {
+      request(app.server)
+        .get('/bundle.js')
+        .expect('X-nonsense-1', 'yes')
+        .expect('X-nonsense-2', 'no')
+        .expect(200, done);
+    });
+  });
```

Comment:
Please rewrite it:
```
const framewroks = [express, fastify];

for (const framework of framewroks) {
  // Tests
}
```

Reply:
@evilebottnawi as fastify don't support `.send` the header `content-length` will be missing from all the tests.